src/commcare_cloud/ansible/roles/formplayer/templates/logback-spring.xml.j2

```diff
@@ -0,0 +1,42 @@
+<configuration>
+
+    <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+        <encoder>
+            <pattern>%d{HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</pattern>
+        </encoder>
+    </appender>
+
+    <appender name="requests" class="ch.qos.logback.core.rolling.RollingFileAppender">
+        <file>request_response.log</file>
+        <!-- Keep a fixed number of logs. -->
+        <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
+            <fileNamePattern>request_response.%i.log</fileNamePattern>
+            <minIndex>1</minIndex>
+            <maxIndex>1</maxIndex>
```

Comment:
Does this mean keep the current log file and one previous one?

Reply:
Yep, this is what I see on my local

<img width="349" alt="Screen Shot 2020-06-22 at 3 15 27 PM" src="https://user-images.githubusercontent.com/615126/85326423-42b04400-b49b-11ea-8f30-c964b8f92e37.png">

Tested with a small `1MB maxFileSize` and saw the rotation. I originally had a window of 3, but figured 3 files of n max size was equivalent to 1 of n*3 for our purposes, since we're going to tail it anyway. They also have some time based rolling policies that log to a file based on the time (daily, weekly, etc.) but when testing this it seemed more prone to uncapped file sizes.